plugins/pulp_rpm/plugins/migrations/0016_new_yum_distributor.py

```diff
@@ -164,8 +165,8 @@ def _re_publish_repository(repo, distributor):
     """
 
     repo = common_utils.to_transfer_repo(repo)
-    repo.working_dir = common_utils.distributor_working_dir(distributor['distributor_type_id'],
-                                                            repo.id)
+    repo.working_dir = distributor_working_dir(distributor['distributor_type_id'],
+                                               repo.id)
```

Comment:
Was the function to do this not moved somewhere else in platform? Or more directly, why is this function in this module instead of importing it from platform as this code used to do?